crates/router/src/core/payments/operations/payment_complete_authorize.rs

```diff
@@ -79,7 +79,7 @@ impl<F: Send + Clone + Sync> GetTracker<F, PaymentData<F>, api::PaymentsRequest>
             .setup_future_usage
             .or(payment_intent.setup_future_usage);
 
-        helpers::authenticate_client_secret(request.client_secret.as_ref(), &payment_intent)?;
+        // helpers::authenticate_client_secret(request.client_secret.as_ref(), &payment_intent)?;
```

Comment:
Please remove the code instead of commenting. Same for the other instances as well.